imap/jmap_contact.c

```diff
@@ -6678,6 +6679,51 @@ static void jscomps_from_vcard(json_t *obj, vcardproperty *prop,
     }
 }
 
+static void vcardvalues_to_json(const char *values, vcardvalue_kind vkind,
+                                json_t *jvals)
+{
+    const char *val, *p = strchr(values, ',');
+    tok_t vals = { 0 };
+
+    if (p && (p == values || p[-1] != '\\')) {
+        tok_init(&vals, values, ",", TOK_EMPTY);
+        val = tok_next(&vals);
+    }
+    else {
+        val = values;
+    }
+
+    do {
+        json_t *jval = NULL;
+
+        switch (vkind) {
+        case VCARD_BOOLEAN_VALUE:
+            jval = json_boolean(!strcasecmp("TRUE", val));
+            break;
+
+        case VCARD_INTEGER_VALUE:
+            jval = json_integer(atoi(val));
```

Comment:
https://datatracker.ietf.org/doc/html/rfc6350#section-4.5 says this is a 64-bit signed integer. If so, we can't use atoi here